### ISSUE TITLE
Fix E2E test: batchMoveModification.spec.ts > Move Operation

### ISSUE THREAD

**Issue by matt-whiteman** [camunda/camunda#18149]
### Description

This test [was identified as flaky](https://docs.google.com/document/d/1HwGJMbgarY7oPgqqoFKj4sWZsRGqvqDG7nI6tKh9fV4/edit) (52% Failure rate)

Example failure: https://github.com/camunda/camunda/actions/runs/9506459081/job/26203708875

```
tests/batchMoveModification.spec.ts:44:7 › Process Instance Batch Modification › Move Operation @roundtrip 

    Error: Timeout 30000ms exceeded while waiting on the predicate

      155 |         {intervals: [2000], timeout: 30000},
      156 |       )
    > 157 |       .toBe(NUM_SELECTED_PROCESS_INSTANCES.toString());
          |        ^
      158 |
      159 |     // Expect that checkPayment flow node instances got canceled in all process instances
      160 |     await expect(

        at /home/runner/_work/camunda/camunda/operate/client/e2e-playwright/tests/batchMoveModification.spec.ts:157:8
```

### TODO

* [ ] investigate root cause
* [ ] fix the issue

### Additional Information
[d8924242b7a8cd25df06a4b5409570b466225311.webm](https://github.com/camunda/camunda/assets/2543743/e0dcf5f5-f452-4a34-9b34-7ed513722857)

---------------------------------------------------------------------------------------------

### Acceptance Criteria

- [ ] Checked if the specific data needs to be provided for the QA process

### Definition of Ready - Checklist

- [ ] The issue has a meaningful title, description and testable acceptance criteria
- [ ] If documentation needs to be updated, an issue is created in the [camunda-docs](https://github.com/camunda/camunda-docs) repo, and the issue is added to our Operate project board.
- [ ] If HELM charts need to be updated, an issue is created in the [camunda-platform-heml](https://github.com/camunda/camunda-platform-helm) repo, and the issue is added to our Operate project board.
- [ ] Potential Security risks have been considered
- [ ] Cross-team dependencies have been considered

**Comment by pedesen** [comment]
Updated this issue and moved `decisionInstances.spec.ts` to a separate issue: https://github.com/camunda/camunda/issues/19372
